cps/tasks/metadata_extract.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, task_message, media_url, original_url, current_user_name):
         self.message = task_message
         self.media_url = media_url
         self.media_url_link = f'<a href="{media_url}" target="_blank">{media_url}</a>'
-        self.original_url = original_url
+        self.original_url = original_url.split("?")[0].replace("media", "meta")
```

Comment:
Can regex be made safer?

(Please test things like... ?)

```suggestion
        self.original_url = original_url.split("?")[0].replace("/media$", "/meta")
```

Reply:
Definitely, but why "$"?

Reply:
Nobody wants corner cases, perpetually exploding in our face! 🤯

So please emphasize code safety — verifying `/media` on the right side of the input string — here's 1 way of doing that:

https://stackoverflow.com/questions/3675318/how-to-replace-some-characters-from-the-end-of-a-string/3675401#3675401

Reply:
> verifying `/media` on the right side of the input string

(Or whatever precondition... that we should be testing for!)